docs/getting_started/install_ez_platform.md

```diff
@@ -122,6 +122,9 @@ After a moment the installer will ask you to provide a few parameters:
     Do not commit the secret to version control systems, or share it with anyone who does not strictly need it.
     If you have any suspicion that the secret may have been exposed, replace it with a new one.
     The same goes for other secrets, like database password, Varnish invalidate token, JWT passphrase, etc.
+    
+    Remember, that after changing the app secret you should regenerate application cache and log out all the users.
+    For more details refer to [Symfony documentation](https://symfony.com/doc/2.8/reference/configuration/framework.html#secret).
```

Comment:
Devil's ~advocate~ support request: "How do I regenerate application cache and log out all the users?"

For the first, we probably have some doc. For the other, do we even have this functionality, or would you have to manually delete all sessions?

Reply:
For the first, maybe rephrasing this to "clear cache" would do the trick. For the second: probably yes, I cannot think of anything automatic for that scenario.